src/kafka/consumer.js

```diff
@@ -259,31 +292,56 @@ class Consumer extends EventEmitter {
       this._consumer = new Kafka.KafkaConsumer(this._config.rdkafkaConf, this._config.topicConf)
 
       this._consumer.setDefaultConsumeTimeout(this._config.options.consumeTimeout)
-      // this._setDefaultConsumeTimeout(this._config.options.consumeTimeout)
+
+      this._consumer.on('warning', warn => {
+        Logger.isDebugEnabled && logger.debug('onWarning', warn)
```

Comment:
I don't believe logger.level(arg1, arg2) will result in what you want. 
I think you need to jam everything into arg1 which is `message`, but you can double check.

here and throughout.